test/arrow/arrow_roundtrip.cpp

```diff
@@ -16,13 +16,11 @@ static void TestArrowRoundtrip(const string &query, bool export_large_buffer = f
 }
 
 static void TestParquetRoundtrip(const string &path) {
-	DuckDB db;
-	Connection con(db);
+	DBConfig config;
+	config.options.allow_unsigned_extensions = true;
```

Comment:
```suggestion
    // This needs to be set since this test will be triggered when testing autoloading
	config.options.allow_unsigned_extensions = true;
```